### ISSUE TITLE
Cannot modify directory name

### ISSUE THREAD

**Issue by kingteng** [mar10/wsgidav#224]
Cannot modify directory name
env: win10+wsl2
wsgidav run on wsl2
run commamd: wsgidav --server ext-wsgiutils -v -H localhost -p 6080 -r /tmp --auth=anonymous
directory name is Chinese  gd18030 ,from 中文 to 中文1

**Comment by mar10** [comment]
Thank you for contributing! :+1:

In order to process your issue, we need some more information, explanation what already was tried and/or concrete data.

**Comment by kingteng** [comment]
1.wsl run command: ```wsgidav --server ext-wsgiutils -v -H localhost -p 6080 -r /tmp --auth=anonymous```
2.win10 cmd mount: ```NET USE * http://localhost:6080/```
3.modify Chinese name folder: from **中文** to **中文1** 
4.wsl console :
``` shell
2021-06-21 10:16:24.289 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp application()...
2021-06-21 10:16:24.290 - <139652983932672> wsgidav                     DEBUG   :  Raising DAVError 404 Not Found: /%E4%B8%AD%E6%96%87
2021-06-21 10:16:24.290 - <139652983932672> wsgidav.error_printer       DEBUG   :  re-raising DAVError(404 Not Found: /%E4%B8%AD%E6%96%87)
2021-06-21 10:16:24.290 - <139652983932672> wsgidav.error_printer       DEBUG   :  caught DAVError(404 Not Found: /%E4%B8%AD%E6%96%87)
2021-06-21 10:16:24.290 - <139652983932672> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-06-21 02:16:24] "PROPFIND /%E4%B8%AD%E6%96%87" length=0, depth=0, connection="Keep-Alive", agent="Microsoft-WebDAV-MiniRedir/10.0.19043", elap=0.001sec -> 404 Not Found
2021-06-21 10:16:24.290 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiStartResponse(404 Not Found, [('Content-Type', 'text/html'), ('Content-Length', '399'), ('Date', 'Mon, 21 Jun 2021 02:16:24 GMT')], None)
2021-06-21 10:16:24.290 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: send headers ''404 Not Found'', [('Content-Type', 'text/html'), ('Content-Length', '399'), ('Date', 'Mon, 21 Jun 2021 02:16:24 GMT')]
2021-06-21 10:16:24.291 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: write 399 bytes: '"<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' "'...
2021-06-21 10:16:24.291 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp finally.
2021-06-21 10:16:30.797 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp application()...
2021-06-21 10:16:30.797 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-1b07f298-85d4-4f75-b41e-8f6c47ac3ce2.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-1eeee54f-0878-4c4d-a724-cef08391c995.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-2a38396a-50cf-4309-bac0-abb08a14fb17.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-git-2762b090c5.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-git-e98116fcef.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-git-1c7a85e4c4.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-4b32dd67-99b1-40d3-b5ff-3e3ba3fdd650.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-git-4d80331859.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-6faa7eff-b632-4d98-b733-ff9a4d6f16dd.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-151381aa-c31f-4512-8731-68ac5956fc08.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-git-b7f28e7e98.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-80844d26-de8a-4eb9-af46-83f3d78cb2e8.sock'
2021-06-21 10:16:30.798 - <139652983932672> wsgidav.fs_dav_provider     DEBUG   :  Skipping non-file '/tmp/vscode-ipc-b47b1d4d-d807-43a6-90f5-416881d0b108.sock'
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.iDxcgX): None
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.iDxcgX): None
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.iDxcgX): None
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.GvYkcE): None
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.GvYkcE): None
2021-06-21 10:16:30.799 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env.GvYkcE): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/model.lock): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/model.lock): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/model.lock): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env-YQHkc5): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env-YQHkc5): None
2021-06-21 10:16:30.800 - <139652983932672> wsgidav                     DEBUG   :  mimetype(/vscode-distro-env-YQHkc5): None
2021-06-21 10:16:30.801 - <139652983932672> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-06-21 02:16:30] "PROPFIND /" length=0, depth=1, connection="Keep-Alive", agent="Microsoft-WebDAV-MiniRedir/10.0.19043", elap=0.004sec -> 207 Multi-Status
2021-06-21 10:16:30.801 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiStartResponse(207 Multi-Status, [('Content-Type', 'application/xml'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT'), ('Content-Length', '10669')], None)
2021-06-21 10:16:30.801 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: send headers ''207 Multi-Status'', [('Content-Type', 'application/xml'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT'), ('Content-Length', '10669')]
2021-06-21 10:16:30.801 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: write 10669 bytes: ''<?xml version="1.0" encoding="utf-8" ?>\n<ns0:multi''...
2021-06-21 10:16:30.801 - <139652983932672> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp finally.
2021-06-21 10:16:30.812 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp application()...
2021-06-21 10:16:30.812 - <139652975539968> wsgidav                     DEBUG   :  Raising DAVError 404 Not Found: /%E4%B8%AD%E6%96%87
2021-06-21 10:16:30.812 - <139652975539968> wsgidav.error_printer       DEBUG   :  re-raising DAVError(404 Not Found: /%E4%B8%AD%E6%96%87)
2021-06-21 10:16:30.812 - <139652975539968> wsgidav.error_printer       DEBUG   :  caught DAVError(404 Not Found: /%E4%B8%AD%E6%96%87)
2021-06-21 10:16:30.812 - <139652975539968> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-06-21 02:16:30] "PROPFIND /%E4%B8%AD%E6%96%87" length=0, depth=0, connection="Keep-Alive", agent="Microsoft-WebDAV-MiniRedir/10.0.19043", elap=0.000sec -> 404 Not Found
2021-06-21 10:16:30.812 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiStartResponse(404 Not Found, [('Content-Type', 'text/html'), ('Content-Length', '399'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT')], None)
2021-06-21 10:16:30.813 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: send headers ''404 Not Found'', [('Content-Type', 'text/html'), ('Content-Length', '399'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT')]
2021-06-21 10:16:30.813 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: write 399 bytes: '"<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' "'...
2021-06-21 10:16:30.813 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp finally.
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp application()...
2021-06-21 10:16:30.822 - <139652975539968> wsgidav                     DEBUG   :  Raising DAVError 404 Not Found: /%E4%B8%AD%E6%96%87/desktop.ini
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.error_printer       DEBUG   :  re-raising DAVError(404 Not Found: /%E4%B8%AD%E6%96%87/desktop.ini)
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.error_printer       DEBUG   :  caught DAVError(404 Not Found: /%E4%B8%AD%E6%96%87/desktop.ini)
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-06-21 02:16:30] "PROPFIND /%E4%B8%AD%E6%96%87/desktop.ini" length=0, depth=0, connection="Keep-Alive", agent="Microsoft-WebDAV-MiniRedir/10.0.19043", elap=0.000sec -> 404 Not Found
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiStartResponse(404 Not Found, [('Content-Type', 'text/html'), ('Content-Length', '411'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT')], None)
2021-06-21 10:16:30.822 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: send headers ''404 Not Found'', [('Content-Type', 'text/html'), ('Content-Length', '411'), ('Date', 'Mon, 21 Jun 2021 02:16:30 GMT')]
2021-06-21 10:16:30.823 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  wsgiWriteData: write 411 bytes: '"<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' "'...
2021-06-21 10:16:30.823 - <139652975539968> wsgidav.server.ext_wsgiutils_server DEBUG   :  runWSGIApp finally.
```

**Comment by mar10** [comment]
Shoulkd be fixed in 4.0
```
17:30:37.436 - <16828> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-11-27 16:30:37] "PROPFIND /测试/中文" length=0, depth=0, elap=0.001sec -> 207 Multi-Status
17:30:37.444 - <16776> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-11-27 16:30:37] "MOVE /测试/中文" dest="http://localhost:8080/%E6%B5%8B%E8%AF%95/%E4%B8%AD%E6%96%871", length=0, depth=infinity, overwrite=F, elap=0.004sec -> 201 Created
17:30:37.455 - <10644> wsgidav.wsgidav_app         INFO    :  127.0.0.1 - (anonymous) - [2021-11-27 16:30:37] "PROPFIND /测试/中文1" length=0, depth=0, elap=0.000sec -> 207 Multi-Status
```
